src/TextShow/Data/VectorNonEmpty.hs

```diff
@@ -0,0 +1,27 @@
+{-# LANGUAGE TemplateHaskell #-}
+{-# OPTIONS_GHC -Wno-orphans #-}
+{-|
+Module:      TextShow.Data.Vector.NonEmpty
+Copyright:   (C) 2014-2023 Ryan Scott
```

Comment:
The copyright year should be 2023, since that's when the code in this module was written. Also, since you were the one who wrote this, you might as well list yourself as the copyright owner.